### ISSUE TITLE
C++ STL Problem

### ISSUE THREAD

**Issue by zahinekbal** [zahinekbal/Contributed-code-in-Hackoctober#3]

**Comment by MonarchChakri** [comment]
Hi, whats the problem statement and what is expected to be solved?

**Comment by zahinekbal** [comment]
> Hi, whats the problem statement and what is expected to be solved?

C++ STL problem if you wanna add Sorting with C++ Stl or any problems then go ahead ....but make sure if you are adding problem in C++ Stl then add the questions with code ..

**Comment by surya7765** [comment]
I want to work on this issue.

**Comment by zahinekbal** [comment]
> I want to work on this issue.

go ahead @surya7765

**Comment by MonarchChakri** [comment]
@zahinekbal Create a PR https://github.com/zahinekbal/codeWith-hacktoberfest/pull/19
Please review and merge.

**Comment by kubercodes** [comment]
hey @zahinekbal  can I work on this issue.

**Comment by zahinekbal** [comment]
yeah this issues is open for all but before making PR make sure Noone is doing the same thing....
after making PR just comment your issues here....

**Comment by kubercodes** [comment]
Hey @zahinekbal I have submitted a PR with the question and its c++ stl solution .
Please review and merge it.

**Comment by surya7765** [comment]
Hey, @zahinekbal I have submitted a PR #42 .
Please review it. 
According to hacktoberfest rule while merging please label it as "hacktoberfest-accepted".

**Comment by sachin-611** [comment]
@zahinekbal will you create a new issue for solutions of cp (codeforces and codechef)?

**Comment by paritoshkray** [comment]
Can I add a question and solution using C++ STL priority queue?

**Comment by zahinekbal** [comment]
I created

**Comment by zahinekbal** [comment]
> Can I add a question and solution using C++ STL priority queue?

Yeahhh sure

**Comment by vaishnavijha** [comment]
I have created a pull request of using comparators with STL in cpp

**Comment by absum11** [comment]
A pull request created.
Please review PR#110.

**Comment by Palak0519** [comment]
I have proposed a solution to demonstrate the function strstr implementation in C++
A pull request is created.
Kindly review PR#157

**Comment by yashgupta18** [comment]
Can I add Dynamic programming questions??

**Comment by harsh1245-bit** [comment]
Please accept #177 PR.
